PyPi_Setup.md

```diff
@@ -0,0 +1,217 @@
+# PyPi setup 
+
+## 1. Introduction
+
+In this guide, We will provide you with a comprehensive overview of how to set up and deploy a Python package to the Python Package Index (PyPI). This package is designed to facilitate the sharing of Python projects, enabling you to install it easily using pip.
+
+### Purpose
+
+The objectives of this guide is to help you to:
+
+- Understand the necessary setup for your package
+- Deploy your package to PyPI
+- Install your package easily on any system
+
+## 2. Pre-requisites
+
+Before you begin, ensure that you have the following tools installed:
+
+### Required Tools
+
+- **Python** (version 3.6 or above)
+- **pip** (Python package manager)
+- **setuptools**, **wheel**, **twine** (for building and uploading packages)
+
+## 3. Installations
+
+To prepare your environment, upgrade pip, setuptools, and wheel:
+
+```bash
+sudo python -m pip install --upgrade pip setuptools wheel
+```
+
+Next, install `tqdm` and `Twine`:
+
+```bash
+sudo python -m pip install tqdm
+sudo python -m pip install --user --upgrade twine
+```
+
+These tools are essential for building and uploading your package.
+
+## 4. Project Structure
+
+```
+sneaky_package/
+│
+├── .github/                              # GitHub-specific files
+│   ├── BUG_report_issue_template.md      # Template for bug reports
+│   ├── Feature_request_issue_template.md # Template for feature requests
+│   ├── changes_in_code_issue_template.md # Template for code change issues
+│   └── pull_request_template.md          # Template for pull requests
+│
+├── package/                              # Main package directory
+│   ├── __init__.py                       # Required for package initialization
+│   ├── clp.py                            # Module for CLP functionality
+│   ├── display.py                        # Module for display-related functions
+│   ├── graph.py                          # Module for graph-related functions
+│   ├── models.py                         # Module containing data models
+│   ├── piechart.py                       # Module for pie chart functionalities
+│   └── test.py                           # Test script for testing package modules
+│
+├── .gitignore                            # Specifies files to ignore in Git
+├── CHANGELOG.md                          # Log of changes made to the project
+├── CODE_OF_CONDUCT.md                    # Code of conduct for contributors
+├── LICENSE                               # License information for the project
+├── README.md                             # Documentation about the project
+└── setup.py                              # Setup script for packaging the project
```

Comment:
Just mention files that is related to setup & deploy.
`.github/`, `.gitignore`, `CHANGELOG.md`, `CODE_OF_CONDUCT.md` files are irrelevant